app/src/hooks/useSwap.ts

```diff
@@ -193,6 +194,15 @@ export const useSwap = () => {
     }
   }, [])
 
+  const getNetworkFeeFiatEstimate = React.useCallback(
+    (network: NetworkInfo) => {
+      return (
+        Number(tokenSpotPrices[network.symbol]) * Number(gasEstimates.gasFee)
+      ).toString()
+    },
+    [tokenSpotPrices, gasEstimates]
+  )
```

Comment:
We probably want this in it's own hook, since gas values are displayed in more places than other swap pieces of swap data

Reply:
Moved to it's own hook! :)